lib/dashboard.js

```diff
@@ -10,35 +10,28 @@ const debounce = require('lodash.debounce');
 const express = require('express');
 
 // Ours
-const configHelper = require('../config');
-const log = require('../logger')('nodecg/lib/dashboard');
-const bundles = require('../bundles');
-const ncgUtils = require('../util');
+const configHelper = require('./config/index');
+const log = require('./logger')('nodecg/lib/dashboard');
+const bundles = require('./bundles');
+const ncgUtils = require('./util/index');
```

Comment:
Oh, weird. I didn't actually add these `/index` endings, that's something that WebStorm just did. 🤔